scalardb-test/src/main/java/kelpie/scalardb/transfer/graphql/TransferProcessor.java

```diff
@@ -64,8 +64,6 @@ public class TransferProcessor extends TimeBasedProcessor {
   private final TwoPhaseCommitTransactionManager twoPhaseCommitTransactionManager;
   private final GraphQL graphql;
   private final int numAccounts;
-  private final boolean isVerification;
-  private final boolean useCompactLog;
```

Comment:
Remove the verification test things because we don't run verification test using this processor.